innovation.game.php

```diff
@@ -7389,6 +7389,49 @@ function stPlayerInvolvedTurn() {
             case "124N1":
                 $step_max = 2; // --> 2 interactions: see B
                 break;
+
+            // id 127, Artifacts age 2: Chronicle of Zuo
+            case "127N1":
+                $players = self::loadPlayersBasicInfos();
+				$min_towers = 100;
```

Comment:
I'm not a fan of the fact that this default value is so low, but my suggestion below will eliminate this anyway!

Reply:
Instead of using a for loop to iterate over the players, lets inline a function here which grabs the min value for us.

```$min_towers = self::getUniqueValueFromDB(self::format("SELECT MIN(player_icon_count_4) FROM player WHERE player_id != {player_id}", array('player_id' => $player_id))```

Given that this query is highly specific and pretty short, I don't think there's much value in refactoring it out into a separate function. Especially since this would be hard to give a concise name.